roamance/src/components/social/post/post-creator.tsx

```
@@ -0,0 +1,427 @@
+            />
+          </div>
+
+          <div className="flex-1">
+            <textarea
+              value={text}
+              onChange={handleTextChange}
+              onClick={() => setIsExpanded(true)}
+              placeholder="Share your travel experience..."
+              rows={isExpanded ? 3 : 1}
+              className="w-full p-3 rounded-xl bg-gray-50 dark:bg-gray-700/50 border border-gray-100 dark:border-gray-700 resize-none focus:outline-none focus:ring-2 focus:ring-purple-500/50 text-gray-800 dark:text-gray-200 placeholder-gray-400 dark:placeholder-gray-500 transition-all"
+            />
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)